### ISSUE TITLE
Make SYSTEM_PACKAGES the default (Trac #1401)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1393]
see #977

* Do away with the SYSTEM_PACKAGES flag.
* By default use system packages, and fall back to I3_PORTS. 
* Create a flag to force the use of **only** I3_PORTS

Migrated from https://code.icecube.wisc.edu/ticket/1401
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:05:42", 
    "description": "see #983\n\n* Do away with the SYSTEM_PACKAGES flag.\n* By default use system packages, and fall back to I3_PORTS. \n* Create a flag to force the use of **only** I3_PORTS", 
    "reporter": "nega", 
    "cc": "david.schultz, olivas", 
    "resolution": "fixed", 
    "_ts": "1547251542149600", 
    "component": "cmake", 
    "summary": "Make SYSTEM_PACKAGES the default", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-15T15:19:45", 
    "milestone": "", 
    "owner": "nega", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151029T10:58:35: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20151209T16:53:39: nega changed milestone from "" to "offline V15-12-00"

**Comment by icecube-trac** [comment]
Trac update at 20151209T22:07:11: 
* nega changed status from "assigned" to "closed"
* nega commented:

> In changeset 2361:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2361"
> make SYSTEM_PACKAGES=ON the default
> 
> SYSTEM_PACKAGES now defaults to TRUE/ON. when set to FALSE/OFF cmake will
> use I3_PORTS and ignore system packages. CVMFS enabled systems will "do the
> right thing" regardless.
> 
> more esoteric tools (those used by online?) may need some tweaking, and
> if so, a ticket should be filed.
> 
> closes #1393
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas commented:

> Milestone offline V15-12-00 deleted


* olivas changed milestone from "offline V15-12-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:05:42: nwhitehorn commented:

> In changeset 170156:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170156"
> make SYSTEM_PACKAGES=ON the default
> 
> SYSTEM_PACKAGES now defaults to TRUE/ON. when set to FALSE/OFF cmake will
> use I3_PORTS and ignore system packages. CVMFS enabled systems will "do the
> right thing" regardless.
> 
> more esoteric tools (those used by online?) may need some tweaking, and
> if so, a ticket should be filed.
> 
> closes #1393
> ```
